sympy/functions/elementary/complexes.py

```diff
@@ -395,8 +395,8 @@ class Abs(Function):
     >>> abs(-x) # The Python built-in
     Abs(x)
 
-    Note that the Python built-in will return either an Expr or int depending on
-    the argument::
+    Note that the Python built-in will return either an Expr or int depending
+    on the argument::
```

Comment:
Yeah, but not all rules in PEP8 are important for everybody.
There are lines in the codebase that are *much* longer than 80 characters, and you can't go and fix them all.
So you'll have to find strategies to deal with overlong lines anyway.
Personally, I tend to fix style on code that I touch for other reasons, but leave the rest alone even if I see it and hate it: PEP8 conformance is desirable,  but being able to know who last changed what semantics is important.